### PR TITLE
admin/users load all users now instead of only teachers

### LINKED ISSUE #418: Show students in admin/users
## Description
Currently, admin/users only show users with role = teacher.
All users should be able to be viewed.

## Acceptance criteria
- [ ] All users load in which corresponding roles.

### PR DESCRIPTION
**Pull request by te4-alexander-nylundgomes**
## ⚙️ What issue did you implement or fix?
Fixes #418 

## 📜 Summary
- Added the ability to view all users in /admin/users instead of only teachers.

## 📝 Type of change

Please delete options that are not relevant.

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Could use some testing

### CODE REVIEW

**Comment by itggot-linus-styren** [comment]
To make integration tests pass, please cherry-pick commits from #434 (23dcfd71bf646111fed2d97c7cc15dedcfbbc183 and 9003ce0cb508cd220c9262a4bf184e3ece6daf68)
```
git fetch --all
git cherry-pick 23dcfd7
git cherry-pick 9003ce0
git push
```

**Comment by itggot-linus-styren** [comment]
Just for some context, the issue that introduced this change is this one: #325

At the time it was not a feature to be able to change roles of students, only staff members.

Perhaps it should be changed so admin can see all users, teachers can see their students and students don't have access to this page.

**Review by te4-alexander-kjellberg — CHANGES_REQUESTED** [review]
Remove (unnecessary..?) commented code on rows 64 -> 71?

File: frontend/src/views/AdminUsersView.vue [hunk 1]
```diff
@@ -61,21 +61,33 @@ export default class AdminUsersVIew extends Vue {
...
-    return convertListToN(UsersModule.allAsArray
-      .filter(user => RoleChecker.isStudentHealth(user))
+  // get staffUsers(): UserCollection {
+  //   return convertListToN(UsersModule.allAsArray
+  //     .filter(user => RoleChecker.isStudentHealth(user))
+  //     .map((user) => {
+  //       user.assignedRoles = RoleChecker.assignedRoles(user).map((number) => { return number.toString(); });
+  //       return user;
+  //     }), 'uid');
+  // }
```
Comment (Te4-Jacob-sandstrom) [inline frontend/src/views/AdminUsersView.vue:71]:
```suggestion
```

**Review by Te4-Jacob-sandstrom — CHANGES_REQUESTED** [review]

**Review by Te4-Jacob-sandstrom — APPROVED** [review]

File: frontend/src/views/AdminUsersView.vue [hunk 2]
```diff
@@ -61,21 +61,25 @@ export default class AdminUsersVIew extends Vue {
    * This function filters out all students from the Usersmodule users
    * and gives the remaining an array with their specified roles.
    */
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/views/AdminUsersView.vue:63]:
```suggestion
```

File: frontend/src/views/AdminUsersView.vue [hunk 3]
```diff
@@ -61,21 +61,25 @@ export default class AdminUsersVIew extends Vue {
...
-  get staffUsers(): UserCollection {
-    return convertListToN(UsersModule.allAsArray
-      .filter(user => RoleChecker.isStudentHealth(user))
+
+  /**
+   * This function loads all the users
+   */
```
Comment (Te4-Jacob-sandstrom) [inline frontend/src/views/AdminUsersView.vue]:
```suggestion
```

File: frontend/src/views/AdminUsersView.vue [hunk 4]
```diff
@@ -61,21 +61,25 @@ export default class AdminUsersVIew extends Vue {
    * This function filters out all students from the Usersmodule users
    * and gives the remaining an array with their specified roles.
    */
-  get staffUsers(): UserCollection {
-    return convertListToN(UsersModule.allAsArray
-      .filter(user => RoleChecker.isStudentHealth(user))
+
+  /**
+   * This function loads all the users
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/views/AdminUsersView.vue:66]:
```suggestion
   * This function returns all users
```

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
I will approve this :+1: :shipit:

**Review by Te4-Jacob-sandstrom — CHANGES_REQUESTED** [review]

File: frontend/src/views/AdminUsersView.vue [hunk 5]
```diff
@@ -58,24 +58,25 @@ export default class AdminUsersVIew extends Vue {
     { text: '', value: 'actions' },
   ];
   /**
-   * This function filters out all students from the Usersmodule users
-   * and gives the remaining an array with their specified roles.
+
+  /**
+   * This function returns all users
    */
```
Comment (Te4-Jacob-sandstrom) [inline frontend/src/views/AdminUsersView.vue:64]:
```suggestion
```

**Review by Te4-Jacob-sandstrom — CHANGES_REQUESTED** [review]

**Review by Te4-Jacob-sandstrom — APPROVED** [review]

**Review by te4-alexander-kjellberg — APPROVED** [review]
Looks good :ok_woman:

File: frontend/src/views/AdminUsersView.vue [hunk 6]
```diff
@@ -58,24 +58,21 @@ export default class AdminUsersVIew extends Vue {
     { text: '', value: 'actions' },
   ];
   /**
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/views/AdminUsersView.vue:60]:
```suggestion
```

Fix for syntax errors

**Pull request merged**